### ISSUE TITLE
Question Regarding ShowDownJS

### ISSUE THREAD

**Issue by sapphireloyalty** [showdownjs/showdown#562]
I'm a developer and am developing an JS application that uses GNU Affero General Public License. I'm not good at  understanding licenses yet. Will the license of this project work with the GNU Affero GPL or are they incompatible?

**Comment by svarlamov** [comment]
Not sure if you resolved this on your own yet, but I've found this site quite useful in the past: http://janelia-flyem.github.io/licenses.html

**Comment by tivie** [comment]
I'm no expert in licenses but I would reckon it isn't incompatible. The license we use is fairly permissive.

As long as you keep the following it's fine:

* Redistributions of source code must retain the above copyright notice,
  this list of conditions and the following disclaimer.

* Redistributions in binary form must reproduce the above copyright
  notice, this list of conditions and the following disclaimer in the
  documentation and/or other materials provided with the distribution.

* Neither the name "Markdown" nor the names of its contributors may
  be used to endorse or promote products derived from this software
  without specific prior written permission.


Basically, showdown's name copyright belongs to John Fraser. Markdown's name copyright belongs to John Gruber. 
Neither the names of John Fraser or John Gruber should be used as endorsers (or programmers, or whatever) of your piece of software.
